src/community/spinglass/clustertool.cpp

```diff
@@ -149,29 +146,20 @@ static igraph_error_t igraph_i_community_spinglass_negative(
  *     configuration model is used. The configuration means that the
  *     baseline for the clustering is a random graph with the same
  *     degree distribution as the input graph.
- * \param gamma Real number. The gamma parameter of the
- *     algorithm. This defines the weight of the missing and existing
- *     links in the quality function for the clustering. The default
- *     value in the original code was 1.0, which is equal weight to
- *     missing and existing edges. Smaller values make the existing
- *     links contibute more to the energy function which is minimized
- *     in the algorithm. Bigger values make the missing links more
- *     important. (If my understanding is correct.)
+ * \param gamma Real number. The gamma parameter of the algorithm,
+ *     acting as a resolution parameter. Smaller values typically lead to
+ *     larger clusters, larger values typically lead to smaller clusters.
```

Comment:
Is this equivalent to `resolution` in Leiden, and if yes, would it make sense to rename it accordingly? Parameter renamings are not API-breaking in C.

Reply:
Let's do this even if it's not exactly equivalent. `resolution` is better in explaining what the parameter does than `gamma`.

Reply:
Hmm, actually, looking at this again: there's a `gamma_minus` parameter in the version of the function that also allows negative weights. Having `resolution_minus` would be awkward and it's not that clear any more once you get to that function that it controls the resolution directly. I propose keeping the name as is.